numba/bytecode.py

```diff
@@ -35,128 +35,6 @@ def get_code_object(obj):
     return getattr(obj, '__code__', getattr(obj, 'func_code', None))
 
 
-def _make_bytecode_table():
```

Comment:
This table is no longer needed.   Originally, this table encodes the length of each instruction.  Python3.6 switched to fixed-length instruction word (16-bit).  It turns out pre-3.6 instruction length can be determined by whether the opcode is >= than `dis.HAVE_ARGUMENT`, which indicates the bytecode has 2-extra bytes for the argument.